nixos/modules/services/misc/nix-daemon.nix

```diff
@@ -127,7 +127,7 @@ in
 
       useSandbox = mkOption {
         type = types.either types.bool (types.enum ["relaxed"]);
-        default = false;
+        default = true;
         description = "
           If set, Nix will perform builds in a sandboxed environment that it
           will set up automatically for each build. This prevents impurities
```

Comment:
"This isn't enabled by default" below should be updated

Reply:
Thanks! I updated the change.